### DISCUSSION TITLE
اضافه کردن API

### DISCUSSION THREAD

**Discussion by shzdiushju** [hiddify/Hiddify-Manager#4218] [Ideas ایدهها]
دروود

درخواست اضافه شدن API جدید برای خواندن و دسترسی به لاگ های Xray را دارم, به دلیل نیاز مانیتور کردن یوزر ها (ایپی و UUID) همانند مرزبان

متشکرم

**Comment by TomCat-Hub** [comment]
> دروود
> 
> درخواست اضافه شدن API جدید برای خواندن و دسترسی به لاگ های Xray را دارم, به دلیل نیاز مانیتور کردن یوزر ها (ایپی و UUID) همانند مرزبان
> 
> متشکرم



> دروود
> 
> درخواست اضافه شدن API جدید برای خواندن و دسترسی به لاگ های Xray را دارم, به دلیل نیاز مانیتور کردن یوزر ها (ایپی و UUID) همانند مرزبان
> 
> متشکرم

درود بر شما دوست عزیز

وقت بخیر

لطفا تمامی درخواست های اضافه شدن فیچر رو ایشو کنید تا تیم توسعه مستقیما از اون ها آگاه بشن و به اون ها رسیدگی کنند.

لینک ایشو:
https://github.com/hiddify/Hiddify-Manager/issues

باتشکر از انتخاب شما

مانا باشید

**Reply by shzdiushju** [reply]
خیلی ممنونم از شما
